heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -26,15 +27,19 @@ import {
   InheritanceType,
   PathResolutionMethod
 } from '@rushstack/heft-config-file';
-import { FileSystem, JsonFile, JsonSchema, Terminal } from '@rushstack/node-core-library';
+import { FileSystem, Import, JsonFile, JsonSchema, Terminal } from '@rushstack/node-core-library';
 
 import { IHeftJestReporterOptions } from './HeftJestReporter';
 import { HeftJestDataFile } from './HeftJestDataFile';
 
 type JestReporterConfig = string | Config.ReporterConfig;
-const PLUGIN_NAME: string = 'JestPlugin';
-const PLUGIN_SCHEMA_PATH: string = `${__dirname}/schemas/heft-jest-plugin.schema.json`;
-const JEST_CONFIGURATION_LOCATION: string = `config/jest.config.json`;
+
+interface IJsonPathMetadataOptions {
+  buildFolder: string;
+  resolveAsModule?: boolean;
+  modulePrefix?: string;
+  ignoreMissingModule?: boolean;
+}
```

Comment:
This interface needs good documentation for each member.  It is the Rosetta stone for understanding all the components of your `jsonPathMetadata`